Tasks/AzureWebPowerShellDeployment/Publish-AzureWebDeployment.ps1

```diff
@@ -1,6 +1,8 @@
 Trace-VstsEnteringInvocation $MyInvocation
 Import-VstsLocStrings "$PSScriptRoot\Task.json"
 
+Write-Warning "'Azure App Service: Classic' task will be deprecated soon. 'Azure App Service Deploy' task will replace 'Azure App Service: Classic' task and the recommendation is to migrate your Build or Release process to use the 'Azure App Service Deploy' task. Refer https://go.microsoft.com/fwlink/?LinkID=613750 for more details." -Verbose
```

Comment:
Does this warning message appear in Release summary? Do you need a -verbose flag in the end?